lib/console.js

```diff
@@ -83,6 +86,8 @@ function createWriteErrorHandler(stream) {
 }
 
 function write(ignoreErrors, stream, string, errorhandler) {
+  if (groupIndent > 0)
+    string = `${'\t'.repeat(groupIndent)}${string}`;
```

Comment:
That's how I did it at first but figured a tab would allow people to configure the indentation without us having to add configuration in our code. You raise a good point. Consistency is good,  I'll change it to two spaces.